tests/vs-ldns.rs

```diff
@@ -0,0 +1,116 @@
+use std::process::Command;
+
+const TEST_ZONE_NAME: &str = "nlnetlabs.nl";
+const LDNS_NSEC3_CMD: &str = "ldns-nsec3-hash";
+const DNST_NSEC3_SUBCMD: &str = "nsec3-hash";
+
+#[ignore = "should only be run if ldns command line tools are installed"]
+#[test]
+fn nsec3_hash() {
+    // Note: ldns-nsec3-hash defaults NSEC3 iterations to 1, while dnst
+    // nsec-hash defaults NSEC3 iterations to 0.
+    assert_cmds_eq(
+        &[LDNS_NSEC3_CMD, TEST_ZONE_NAME],
+        &[DNST_NSEC3_SUBCMD, "--iterations", "1", TEST_ZONE_NAME],
```

Comment:
I'm not sure about how useful these tests are when the defaults are different? It would make more sense to run this with the ldns compatibility mode I suppose.

Reply:
I think both make sense, otherwise we don't have a complete sanity check of the `dnst` command as a black box (though we do have that in the packaging workflow sanity check).